guides/doc-Configuring_Load_Balancer/topics/Promoting_SCAP_Content_to_Clients.adoc

```diff
@@ -42,7 +42,7 @@ Do not run the Puppet agent on every client manually because the Puppet agent ru
 .Additional Resources
 
 ifdef::satellite[]
```

Comment:
Is this `ifdef` still correct? Now that it's actually a guide within upstream (and also Orcharhino I think, but @maximiliankolb can answer that), this will be useful for everyone.

Reply:
Yes, it is. We might need to fine tune these ifdefs for upstream, actually, because we've decided to use different titles for this guide.

Reply:
You can remove this ifdef now. @Lennonka

Reply:
Oki, thanks. Done.